concepts/05 UI Components/TextArea/00 Getting Started with TextArea/20 Handle Value Change.md

```diff
@@ -0,0 +1,93 @@
+When users type text into the TextArea, the UI component writes this text into the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property when the <a href="https://developer.mozilla.org/en/docs/Web/Events/change" target="_blank">change</a> event is raised. To specify another value change event, use the [valueChangeEvent](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#valueChangeEvent) property.
```

Comment:
I removed the repetition of the word "when", shortened the text a bit, and got rid of the passive in the end.

```suggestion
If a user edits the text in a TextArea, the UI component updates the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. The update happens on the <a href="https://developer.mozilla.org/en/docs/Web/Events/change" target="_blank">change</a> event. To specify another value change event, use the [valueChangeEvent](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#valueChangeEvent) property.
```